### PR TITLE
FORTECH-96 MediaSlider for MediaPlayer

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
- changing the slider skips the playback of the current media to the time point indicated by the slider knob

Enact-DCO-1.0-Signed-off-by: Daniel Stoian daniel.stoian@lgepartner.com

### CODE REVIEW

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM 😉

**Review by ybsung — APPROVED** [review]
LGTM

File: MediaPlayer/MediaPlayer.js [hunk 1]
```diff
@@ -331,7 +352,8 @@ const MediaPlayerBehaviorDecorator = hoc((config, Wrapped) => { // eslint-disabl
 				currentTime: el.currentTime,
 				duration: el.duration,
 				loop: el.loop,
-				paused: el.paused
+				paused: el.paused,
+				proportionPlayed: el.proportionPlayed
```
Comment (MikyungKim) [inline MediaPlayer/MediaPlayer.js:356]: Let's give a fallback value as 0.
```suggestion
				proportionPlayed: el.proportionPlayed || 0
```

Comment (daniel-stoian-lgp) [reply MediaPlayer/MediaPlayer.js]: thanks. the change is pushed

**Comment by MikyungKim** [comment]
👍

**Pull request merged**